src/js/tech/tech.js

```diff
@@ -437,6 +437,15 @@ class Tech extends Component {
     return '';
   }
 
+  static isTech(component) {
+    /* jshint proto:true */
+    let proto = component.__proto__;
+    /* jshint proto:false */
+    if (Object.getPrototypeOf) {
```

Comment:
There isn't. I made this method so that we could generalize the loop above but make sure that Techs aren't getting added as children.
Though, clearly I'm having issues with browser support :)